examples/close_cookie_dialog/close_cookies.py

```diff
@@ -0,0 +1,45 @@
+"""This example demonstrates how to close popup windows (like promotion form) with AgentQL."""
+
+import time
+
+from agentql.ext.playwright.sync_api import Page
+from playwright.sync_api import sync_playwright
+
+# Set the URL to the desired website
+URL = "https://gov.uk/"
+
+QUERY = """
+{
+    cookies_form {
+        reject_btn
+    }
+}
+"""
+
+
+def main():
+    with sync_playwright() as playwright:
+        browser = playwright.chromium.launch(headless=False)
+
+        # Create a new AgentQL page instance in the browser for web interactions
+        page: Page = browser.new_page()
```

Comment:
We may need to disable the pylance warning here by adding a `# type: ignore` at the end of this line. You could ignore other pylance warnings as they will be fixed in the latest version of SDK.

Reply:
This was a copy-pasta of another example... So unless we're updating everywhere, I'll stay with it :)